bindings/python/src/tokenizer.rs

```diff
@@ -687,6 +687,19 @@ impl PyTokenizer {
                             .into_pyerr::<exceptions::PyValueError>()),
                         }?
                     }
+                    "direction" => {
+                        let value: &str = value.extract()?;
+                        params.direction = match value {
+                            "left" => Ok(TruncationDirection::Left),
+                            "right" => Ok(TruncationDirection::Right),
+                            other => Err(PyError(format!(
+                                "Unknown `direction`: `{}`. Use \
+                                 one of `left` or `right`",
```

Comment:
I think we should keep the wording coherent between all our invalid direction errors. I like the fact you added the correct values to the err msg.

Could you update the other direction error messages too ?